### PR TITLE
Completed javascript-functions. Vick Kahlon

### PR DESCRIPTION
**Pull request by vkahlon**
functions

Heads up, I believe I did the objects section correctly, but due to function scope, it wont print with the rest of the output. therefore I did a bandaid solution with a console.log for the screenshot, however, at the bottom of the screenshot, you can see that I did the return function and it worked as intended.

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+function getArea(width, height) {
+  var area = width * height;
+  console.log(area);
+}
+getArea(17, 42);
+
+function getFirstName(person) {
+  var frenchName = person;
+  console.log(frenchName.firstName);
+  return frenchName.firstName;
```
Comment (codingcodymiller) [inline javascript-functions/main.js:22]: You can just `return person.firstName`

File: javascript-functions/main.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  console.log(seconds);
```
Comment (codingcodymiller) [inline javascript-functions/main.js:3]: You shouldn't `console.log` the output inside the function, you should `return` the final output of the function, and you can log it where the function is called.

File: javascript-functions/main.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+convertMinutesToSeconds(5);
+
+function greet(name) {
+  var greeting = 'Hey, ' + name;
+  console.log(greeting);
```
Comment (codingcodymiller) [inline javascript-functions/main.js:9]: You shouldn't `console.log` the output inside the function, you should `return` the final output of the function, and you can log it where the function is called.

File: javascript-functions/main.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+}
+greet('Beavis');
+
+function getArea(width, height) {
+  var area = width * height;
+  console.log(area);
```
Comment (codingcodymiller) [inline javascript-functions/main.js:15]: You shouldn't `console.log` the output inside the function, you should `return` the final output of the function, and you can log it where the function is called.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by vkahlon** [comment]
Hopefully this fixes my beginner mistak
Screen Shot 2021-12-07 at 4 09 09 PM
e!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**